assets/model_monitoring/components/data_drift/data_drift_compute_metrics/spec.yaml

```diff
@@ -29,6 +29,12 @@ inputs:
     type: number
   numerical_threshold:
     type: number
+  override_numerical_feature:
+    type: string
```

Comment:
Is it a formatted string that we can parse multiple feature override entries from it?

Reply:
yes I just followed the pattern of this categorical_metrics. It is a comma-joint string. For example, if user wants to override UserId and TransactionId as numerical value. This will be "UserId,TransactionId".

https://github.com/Azure/azureml-assets/blob/4667dcbbda9316906a718c85b7f878873b3523b2/assets/model_monitoring/components/data_quality/data_quality_compute_metrics/spec.yaml#L22C7-L22C7